### PR TITLE
Toggle showing csp courses

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -480,6 +480,7 @@ def publish_discourse_stylesheet
     DiscourseStylesheets.cache.clear
   end
 
+
```
Comment (Hamms) [inline app/models/category.rb]: nit: extra whitespace

File: app/assets/javascripts/discourse/templates/discovery/categories.hbs [hunk 2]
```diff
@@ -22,16 +22,16 @@
...
                   {{{c.description_excerpt}}}
                 </div>
+                {{#if c.category_expanding_class}}
+                    <button class="btn"
+                      {{action "expand" c.category_expanding_class}}
+                      {{bind-attr id="c.category_expanding_class"}}
+                    >
+                        Expand units for this course
+                    </button>
+                {{/if}}
```
Comment (Hamms) [inline app/assets/javascripts/discourse/templates/discovery/categories.hbs:32]: won't this button be within the very `tr` that it's supposed to be able to show? And therefore itself be hidden?

Comment (mehalshah) [reply app/assets/javascripts/discourse/templates/discovery/categories.hbs:32]: No -`category_expanding_class` gets set if you are the parent (slug name is csp or csd). `category_collapse_class` gets set if youare the child (slug name begins with csp or csd and has numbers after it)

Comment (Hamms) [reply app/assets/javascripts/discourse/templates/discovery/categories.hbs:32]: oh, of course!

**Review by Hamms — APPROVED** [review]

**Pull request merged**